manifests/init.pp

```diff
@@ -102,14 +102,23 @@
   anchor{'pgbouncer::begin':}
 
   # Same package name for both redhat based and debian based
-  package{ $pgbouncer_package_name:
-    ensure  => installed,
-    require => [
-      Class[
-        'postgresql::repo::yum_postgresql_org'],
-        Anchor['pgbouncer::begin']
-      ],
-  }
+ 
+  case $::osfamily {
```

Comment:
Please add a default of failure to this case statement, and close it. (Right now its missing the closing brace and thus failing to parse)